sections/semantics-embedded-content.include

```diff
@@ -3623,27 +3623,58 @@
   For images that have captions, markup generators should use the
   <{figure}> and <{figcaption}> elements to provide the image's caption.
 
-  As a last resort, implementors should either set the <code>alt</code> attribute to the empty string, under
-  the assumption that the image is a purely decorative image that
-  doesn't add any information but is still specific to the surrounding
-  content, or omit the <code>alt</code> attribute
-  altogether, under the assumption that the image is a key part of the
-  content.
+  As a last resort, implementors
+  <a href="https://www.w3.org/TR/ATAG20/#gl_b23">must omit the alt attribute altogether</a>.
+  [[!ATAG20]]
+
+  <div class="note">
+    Setting an <{image}> element's <{img/alt}> attribute to the empty string means the image in
+    question provides no essential information. Assistive technology, such as screen readers,
+    will completely ignore the presence of such an image, as ignoring it will not stop the user
+    from understanding the document, and saves the user time.
+
+    In the following example, the <{img/alt}> is set to the empty string, so it will be silently
+    ignored by screen readers.
+
+    <xmp highlight="html" class="example">
+      <img src="bar-graph.png" alt="">
+    </xmp>
+
+    Unless the image is truly decorative, implementors setting an image's <code>alt</code>
+    attribute to the empty string is not recommended. An empty string will indicate to content
+    management software and accessibility checking software that the image does not need
+    alternative text, and therefore they will not flag the image in question as a potential
+    problem, meaning it is less likely to be repaired.
+
+    In contrast, if an image is vital to understanding the contents of a document, implementors
+    should be aware that an image without an <{img/alt}> attribute is preferable to an
+    <{img/alt}> attribute set to the empty string.
+
+    Taking the previous example and omitting the <code>alt</code> attribute, as such:
```

Comment:
As such reads strange to me. I would find "as follows" or something like that more natural. But this is editorial, feel free to ignore.